src/module/apps/compendium-browser/tabs/action.ts

```diff
@@ -7,7 +7,7 @@ import { ActionFilters, CompendiumBrowserIndexData } from "./data.ts";
 
 export class CompendiumBrowserActionTab extends CompendiumBrowserTab {
     tabName: ContentTabName = "action";
-    tabLabel = "PF2E.Item.Ability.Plural";
+    tabLabel = "PF2E.CompendiumBrowser.TabAction";
```

Comment:
These are in fact _abilities_, not just actions. Quite a few are passive.

Reply:
That string already exists on the system, I've changed it to "Abilities" [here](https://github.com/foundryvtt/pf2e/pull/20160/files#diff-488d88d8d2c417a9ebb2273d262f1f90a447232007d4a2ee5635d6fba79da8b5R1057).

This is meant to avoid having an orphan `TabAction` string and keeping the naming pattern for the strings.

Reply:
Replace it with `TabAbilities`